enterprise/admin/__init__.py

```diff
@@ -1214,3 +1214,36 @@ class EnterpriseGroupMembershipAdmin(admin.ModelAdmin):
         'enterprise_customer_user',
         'pending_enterprise_customer_user',
     )
+
+
+@admin.register(models.LearnerCreditEnterpriseCourseEnrollment)
+class LearnerCreditEnterpriseCourseEnrollmentAdmin(admin.ModelAdmin):
+    """
+    Django admin model for LearnerCreditEnterpriseCourseEnrollmentAdmin.
+    """
```

Comment:
There was previously no way to observe these records (or toggle the revoked state) via the django admin site.